t2t/nlevel.py

```diff
@@ -553,6 +560,7 @@ def set_preliminary_name_and_rank(tree):
 
 def make_consensus_tree(cons_split, check_for_rank=True, tips=None):
```

Comment:
This method is actually redundant at this time as `TreeNode` has a `from_taxonomy` method, but I don't know off hand how easy it would be to drop in here. Just making a general comment, not blocking